Gemfile

```diff
@@ -1,6 +1,6 @@
 source 'https://rubygems.org'
 
-ruby "2.2.3"
+ruby "2.3.0"
```

Comment:
Cool. Maybe we could switch from CircleCI to Travis and utilize the ruby matrix? Support Ruby 2.2, 2.3, and head?